lib/managers/KalturaAdIntegrationManager.js

```diff
@@ -830,7 +830,9 @@ KalturaAdIntegrationManager.prototype.sendBeacon = function(request, response, p
 		};
 
 		for(var i=0; i < events.length; i++){
-			httpGet(events[i].trim(), eventType);
+			var url = events[i];
+			if (url && typeof(url) == 'string' && url.startsWith('http'))
+				httpGet(events[i].trim(), eventType);
```

Comment:
I think node string does not have startsWith method, You should use a different method here or else an exception will be raised